src/corehost/common/pal.windows.cpp

```diff
@@ -27,7 +26,19 @@ bool pal::find_coreclr(pal::string_t* recv)
         }
     }
 
-    // TODO: Try somewhere in Program Files, see https://github.com/dotnet/cli/issues/249
+
+    // Try %ProgramFiles%. Note this works for both x86 and x64/wow64 as per:
```

Comment:
Yeah. This comment helps how it works for x86/x64/wow64. Not sure about nano server? Last ditch, if unset, fail.